### DISCUSSION TITLE
Is there a way to execute the hook script after the execution of CocInstall is complete?

### DISCUSSION THREAD

**Discussion by arashrun** [neoclide/coc.nvim#5044] [Q&A]
I want to compile the native add-ons in the specified dependency packages when I download my plugins via `CocInstall`. Adding postinstall doesn't work because coc default install uses the `--ignore-scripts` option.

**Comment by fannheyward** [comment]
Try `:CocInstall https://github.com/arashrun/coc-mantic`
